release/rllib_tests/learning_tests/yaml_files/a-e/cql-halfcheetahbulletenv-v0.yaml

```diff
@@ -3,7 +3,8 @@ cql-halfcheetahbulletenv-v0:
     run: CQL
     pass_criteria:
         evaluation/episode_reward_mean: 400.0
-        timesteps_total: 10000000
+        # Can not check throughput for offline methods.
```

Comment:
But what about training throughput?
Now that we are consequently distinguishing these counters via the `training_step` API, we should be able to do something like:
```
    num_env_steps_trained: 10000000
```

Reply:
yeah totally. the problem today is everything is hardcoded, so you'd have to update run_learning_tests_from_yaml() to look for num_env_steps_trained, and compute a different throughput, etc.
and then, we need to figure out a good throughput to use for all these algorithms.
when I tested things, our throughput wasn't great. some of the algorithms even when they pass the reward bar, their throughput is significantly lower than the specified threshold.
feel like this can be a TODO?